src/app/public/plugin-resources/code-examples/inline-form/basic/inline-form-demo.component.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  Component
+} from '@angular/core';
+
+import {
+  SkyInlineFormButtonLayout,
+  SkyInlineFormCloseArgs,
+  SkyInlineFormConfig
+} from '@skyux/inline-form';
+
+@Component({
+  selector: 'app-inline-form-demo',
+  templateUrl: './inline-form-demo.component.html'
+})
+export class InlineFormDemoComponent {
+
+  public demoModel: {
+    firstName?: string;
+  } = { };
+
+  public inlineFormConfig: SkyInlineFormConfig = {
+    buttonLayout: SkyInlineFormButtonLayout.SaveCancel
+  };
+
+  public showForm: boolean = false;
+
+  public firstName: string = 'Jane';
```

Comment:
Check your alphabetical ordering here. Looks like the other example needs a fix too.